src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.test.ts

```diff
@@ -152,6 +161,21 @@ describe('WorkspaceSavedObjectsClientWrapper', () => {
         await wrapper.delete(...deleteArgs);
         expect(clientMock.delete).toHaveBeenCalledWith(...deleteArgs);
       });
+      it('should call client.delete if user/groups is dashboard admin', async () => {
+        const {
+          wrapper,
+          clientMock,
+          permissionControlMock,
+          requestMock,
+        } = generateWorkspaceSavedObjectsClientWrapper(DASHBOARD_ADMIN);
+        expect(getWorkspaceState(requestMock)).toEqual({
+          isDashboardAdmin: true,
+        });
+        const deleteArgs = ['dashboard', 'not-permitted-dashboard'] as const;
+        await wrapper.delete(...deleteArgs);
+        expect(permissionControlMock.validate).not.toHaveBeenCalled();
+        expect(clientMock.delete).toHaveBeenCalledWith(...deleteArgs);
+      });
```

Comment:
can we combine those cases into one and verify wrapper is same as clientMock?

Reply:
Yes, I have combined these cases related to dashboard admin.